packaging/hudi-flink-bundle/pom.xml

```diff
@@ -538,8 +587,16 @@
 
   <profiles>
     <profile>
-      <id>flink-bundle-shade-hive</id>
+      <id>flink-bundle-shade-hive2</id>
```

Comment:
should we build the bundle against different hive versions? Is that possible to put all hive sync related stuff in the `hive-sync` package so we don't have to maintain different set up across different compute engine?

Reply:
@garyli1019 We can do the refactor in another PR if you think it is necessary.

Reply:
I think we should manage the hive version at the project level, like spark and scala. We can do the refactoring in a separate PR.